grakn-graql/src/main/java/ai/grakn/graql/internal/reasoner/atom/binary/RelationAtom.java

```diff
@@ -475,7 +466,7 @@ private RelationAtom inferRelationType(Answer sub){
     }
 
     @Override
-    public Atom inferTypes() {
+    public RelationAtom inferTypes() {
```

Comment:
Maybe this should be `RelationshipAtom`?

Reply:
naaaaah